### PR TITLE
eslinter warning clean-up

### PR DESCRIPTION
**Pull request by dhuangg**
cleaned up a ton of warnings. still a work in progress. was also able to round out the material ui tabs :)

### CODE REVIEW

File: src/components/FeatureViews/Scheduler/EditSessionView.js [hunk 1]
```diff
@@ -56,7 +56,9 @@ const EditSessionView = ({course, session, editSelection}) => {
     useEffect(() => {
         if (course && session) {
             let durationHours = Math.abs(session.end_datetime - session.start_datetime) / 36e5;
-            durationHours === 0 ? durationHours = 1 : durationHours = durationHours;
+            if (durationHours === 0) {
+                durationHours = 1
```
Comment (NeilAllavarpu) [inline src/components/FeatureViews/Scheduler/EditSessionView.js]: Semicolon

File: src/components/Form/Form.js [hunk 2]
```diff
@@ -223,6 +223,7 @@ class Form extends Component {
                                             NewState[title][name] = id;
                                             break;
```
Comment (NeilAllavarpu) [inline src/components/Form/Form.js]: Remove unnecessary `break` here.

File: src/components/Form/TutoringPriceQuote.js [hunk 3]
```diff
@@ -36,39 +27,27 @@ const TutoringPriceQuote = ({courseType, handleUpdatePriceFields,  tutoringCateg
...
 
     useEffect(()=>{
-        if(requestStatus.priceRule[GET][REQUEST_ALL] === 200 &&
-        requestStatus.category[GET] === 200){
+        if (hooks.isSuccessful(categoryStatus, priceRuleStatus)) {
             setPriceRules(adminPriceRules
                 .filter(rule => rule.course_type === courseType)
                 .map((rule) =>({
                 ...rule,
                 category: categories.find((category) => {return category.id === rule.category}),
```
Comment (NeilAllavarpu) [inline src/components/Form/TutoringPriceQuote.js]: `find` parameter can be simplified to `({id}) => id === rule.category`

File: src/reducers/calendarReducer.js [hunk 4]
```diff
@@ -13,32 +12,26 @@ export default function Calendar(state = initialState.CalendarData, { payload, t
...
-            const filter_value = payload.value
-
+            const filter_key = payload.key;
+            const filter_value = payload.value;
 
             // Search by subject
             newState.events_in_view.filter((allCourse) => {
-                return allCourse[filter_key] === filter_value
+                return allCourse[filter_key] === filter_value;
             })
```
Comment (NeilAllavarpu) [inline src/reducers/calendarReducer.js]: `filter` parameter can be simplified to `(allCourse) => allCourse[filter_key] === filter_value`

**Comment by dhuangg** [comment]
I rebased instead of merged your changes lol

File: src/components/Form/Form.js (hunk 2 above)
Comment (dhuangg) [reply src/components/Form/Form.js]: Eslinter was giving a warning for this lol

File: src/reducers/calendarReducer.js (hunk 4 above)
Comment (dhuangg) [reply src/reducers/calendarReducer.js]: How did this even show up as a change 😶

File: src/components/Form/Form.js (hunk 2 above)
Comment (NeilAllavarpu) [reply src/components/Form/Form.js]: You have a break on the next line right outside the `if` statement, so an extra break inside the `if` is unnecessary

**Pull request merged**